crypto/ec/curve448/word.h

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2014 Cryptography Research, Inc.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Originally written by Mike Hamburg
+ */
+
+#ifndef __WORD_H__
```

Comment:
not sure if we should use double underscore
i.e. reserved identifier for header inclusion macros.
I see other headers use a HEADER_ prefix for this.